packages/devextreme/js/viz/core/base_widget.d.ts

```diff
@@ -32,6 +32,7 @@ import {
     TextOverflow,
     Theme,
     WordWrap,
+    Font as CommonFont,
```

Comment:
We can avoid using renaming here:
```suggestion
    Font,
```
I've tried it locally - seems to work fine:


```ts
/**
 * @namespace DevExpress.viz
 * @deprecated Use Font from DevExpress.common.charts.Font instead
 */
export type Font = Font;
```

Reply:
And you'll have to add a line to the aliases.d.ts file with different deprecation message

Reply:
Nope, this doesn't work. Behaves as expected, there is no magic in this world(